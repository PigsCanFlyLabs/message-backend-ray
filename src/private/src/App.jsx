import '@src/styles/global.scss';

import React from 'react';
import { BrowserRouter, Routes, Route } from 'react-router-dom';

import { Provider } from 'react-redux'

import { ProviderComposer } from '@src/components';
import { ErrorBoundary, Page200, Page404, Page500 } from '@src/pages';
import store from '@src/store';

const PROVIDERS = [
  [Provider, { store }],
];

export default function App() {
  return (
    <ErrorBoundary>
      <BrowserRouter>
        <ProviderComposer providers={PROVIDERS}>
        <Routes>
          <Route path="/" name="index" element={<Page200 />} />
          <Route path="404" name="404" element={<Page404 />} />
          <Route path="500" name="500" element={<Page500 />} />
          <Route path="*" element={<Page404 />} />
        </Routes>
        </ProviderComposer>
      </BrowserRouter>
    </ErrorBoundary>
  );
}
