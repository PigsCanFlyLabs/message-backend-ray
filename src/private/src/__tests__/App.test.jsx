import React from 'react';

import App from '../App';

describe('App', () => {
  it('should render without errors', () => {
    const component = <App />;
    expect(component).toMatchSnapshot();
  });
});
