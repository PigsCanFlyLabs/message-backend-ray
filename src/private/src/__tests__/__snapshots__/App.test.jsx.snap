// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without errors 1`] = `<App />`;
